src/app/jobr/Taschenrechner.java

```diff
@@ -0,0 +1,66 @@
+package app.jobr;
```

Comment:
Der Taschenrechner müsste nicht mehr im Pull Request sein, vll noch einmal git rebase

Reply:
also dafür solltest du einen rebase des testing branches machen
Mit IDE support sollte das relativ leicht gehen.
* Aktuellen stand des Master lokal holen
* den Testing Branch auschecken
* dann ``git rebase master`` damit spielt du die Änderungen des Testing-Branches oben auf die letzten  Änderungen des Master-Branches drauf
* Das Ergebnis dann kontrollieren und force pushen

Zur Sicherheit kannst du dir auch den Testing branch in einen neuen Branch ziehen, dann hast du im Zweifel ein Backup

Reply:
Scheint nicht zu funktionieren, vielleicht kannst du es auch einmal ausprobieren?